options/locale/locale_en-US.ini

```diff
@@ -2525,6 +2525,7 @@ form.create_org_not_allowed = You are not allowed to create an organization.
 settings = Settings
 settings.options = Organization
 settings.full_name = Full Name
+settings.email = Contact E-Mail
```

Comment:
https://en.wikipedia.org/wiki/Email

Other strings also use "email" in en-US.ini